public/index.html

```diff
@@ -30,8 +30,11 @@
     -->
     <script>
       const hostnameParts = window.location.hostname.split('.');
-      const subdomain = hostnameParts[0];
-      const manifestUrl = `%PUBLIC_URL%/manifest.${subdomain}.json`;
+      let subdomain = hostnameParts[0]
+      if (window.location.hostname.includes("localhost")) {
```

Comment:
If we want sites to be accessible at `https://[sitename].localhost:3000` then the subdomain should not set to default when localhost is present

Reply:
Nice catch. Updated.

Reply:
Another update on this, after discussion with Rohit, looks like we do need the subdomain set to `default` on localhost since that lets us run the application locally without copying over the files from the config repo.
I've updated the code in the `install-prompt.tsx` as well to reflect these changes.